src/components/tasks/card/index.tsx

```diff
@@ -31,7 +29,7 @@ import { useEditMode } from '@/hooks/useEditMode';
 import { useGetUsersByUsernameQuery } from '@/app/services/usersApi';
 import { ConditionalLinkWrapper } from './ConditionalLinkWrapper';
 import { isNewCardDesignEnabled } from '@/constants/FeatureFlags';
-
+import useUserData from '@/hooks/useUserData';
 type Props = {
     content: task;
     shouldEdit: boolean;
```

Comment:
please put this type in types file and export it from there

Reply:
This prop type has conditional types written based on the props passed to that component. I think it is better to keep them there only.